frontend/public/components/_nav.scss

```diff
@@ -7,24 +7,25 @@ $active-border: 3px;
 .co-m-nav-link {
   display: block;
   float: none;
-  font-size: 13px;
+  font-size: 12px;
   line-height: 23px;
   overflow-x: hidden;
   padding: 2px 0 2px $link-padding-left;
   white-space: nowrap;
 
   a {
     background-color: inherit;
-    color: $color-os-nav-title;
+    color: $color-pf-black-300;
     display: inline-block;
     text-decoration: none;
     width: 100%;
   }
 
   &.active {
+    background-color: $color-pf-black-700;
+    border-left: $active-border solid $color-pf-blue-300;
+    font-weight: 600;
     padding-left: $link-padding-left - $active-border;
-    background-color: #383F47;
-    border-left: $active-border solid $color-os-nav-border-left-active;
```

Comment:
`$color-os-nav-border-left-active` is set to `$color-pf-blue-300`.  No change needed.